src/Microsoft.HttpRepl/OpenApi/OpenApiV3EndpointMetadataReader.cs

```diff
@@ -36,6 +36,15 @@ public IEnumerable<EndpointMetadata> ReadMetadata(JObject document)
 
                         if (method.Value is JObject methodBody)
                         {
+                            if (!(methodBody["responses"] is JObject))
```

Comment:
nit: Do you think it will be useful to add the links that you shared in the pr description here for future reference?

Reply:
Added it in https://github.com/aspnet/HttpRepl/pull/103/commits/f19bd21274b4886d7153e60444bc1f17349da915